config-schema.json

```diff
@@ -47,19 +49,21 @@
                 "$id": "#/properties/entities/properties/light/items/properties/friendly_name",
                 "type": "string",
                 "title": "The Friendly_name Schema",
+                "default": null,
                 "examples": [
-                  "Előszoba lámpa"
+                  "Hallway light"
                 ],
-                "pattern": "^(.*)$"
+                "pattern": ""
```

Comment:
Empty pattern makes no sense. Must be removed, otherwise the validation might fail depending on the used library!
Applies to all pattern definitions in the schema.